### PR TITLE
add checker for non relative links

### PR DESCRIPTION
**Pull request by joshmarinacci**

### CODE REVIEW

**Comment by EliSchleifer** [comment]
Check is failing
https://github.com/trunk-io/.trunk/actions/runs/10605073045/job/29393111842

**Comment by joshmarinacci** [comment]
> Check is failing https://github.com/trunk-io/.trunk/actions/runs/10605073045/job/29393111842

Strange.  I copied this from Tyler's blog:  https://trunk.io/blog/fixme-please-an-exercise-in-todo-linters#grep-linter-

**Review by gewenyu99 — APPROVED** [review]

File: find_nonrelative.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+LINT_TARGET="${1}"
+TODO_REGEX="docs.trunk.io"
```
Comment (EliSchleifer) [inline find_nonrelative.sh:4]: TODO_REGEX doesn't make sense here as a variable name - COPY PASTA

**Review by EliSchleifer — COMMENTED** [review]
copy pasta issue

**Pull request merged**